src/pages/CogPage.vue

```diff
@@ -85,9 +96,33 @@ export default class CogPage extends Vue {
           return;
         }
       }
+
       const resp = await fetch(`${c.COGS}/${params.user}/${params.repo}/${params.cog}`);
       const json = await resp.json();
       this.cog = json.results;
+
+      // finding related cogs based on tags
+      const relres = this.cog.tags.map(async (tag) => {
```

Comment:
not a fan of making a ton of requests here tbh, maybe take just one tag here for now? (this whole related thing might be better off on the backend in the future)...

I would even go as far as saying - maybe then just rework this whole thing to fetch all the cogs and then just filter them here instead of requesting just one? idk... 

Easy temp solution tho - would be just to limit it to one tag, because I'm not a fan of doing a ton of requests just to get the relative stuff.